tests/phpstan/phpstan-1.7.5.1.neon

```diff
@@ -0,0 +1,8 @@
+includes:
+	- %currentWorkingDirectory%/tests/phpstan/phpstan.neon
+
+parameters:
+	ignoreErrors:
+		- '#Call to function is_array\(\) with Currency will always evaluate to false\.$#'
```

Comment:
If you include `phpstan.neon` in this file, the errors you already ignored may be removed here.

Reply:
No. The inheritance works as errors as defined in tests/phpstan/phpstan.neon  are everywhere.